roles/programming-langs/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+# tasks file for programming-langs
+
+- name: Install Ruby packages
+  apt: name={{ ruby_packages }} state=latest
+
+  # update ruby to latest stable version, as lam will generally use the same
+- name: Update Ruby with the command module
+  command: rvm get stable
```

Comment:
I guess I misunderstood this when talking yesterday. I think rvm sets up a weird chain of dependencies if it's not in the apt sources. Line 5 above will install the Ubuntu Ruby version, then to run this, we'd have to pull RVM from outside the repositories. Once RVM's there, I'm assuming this would install a 2nd copy of Ruby, potentially at a higher version.

We should probably stick to either apt ruby or rvm ruby, but not both. My gut says apt ruby, unless we have a compelling reason to go outside the official repositories.